apps/client/src/routes/dashboards/dashboards-index/components/dashboard-table-header/create-dashboard-button.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from '@cloudscape-design/components/button';
+import { FormattedMessage } from 'react-intl';
+
+import { CREATE_DASHBOARD_HREF } from '~/constants';
+import { useApplication } from '~/hooks/application/use-application';
+
+/** Button used to navigate to the create dashboard page. */
```

Comment:
Is this comment necessary? :)

Reply:
hehe I think so. It reveals the navigation nature of the button :p